libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -207,6 +238,13 @@ def _convert_delta_to_message_chunk(
         )
     elif role or default_class == ChatMessageChunk:
         return ChatMessageChunk(content=content, role=role, id=id_)
+    elif role or default_class == AIToolCallsMessageChunk:
```

Comment:
think we want to check for role == "tool_calls" here

Reply:
From what I've seen we enter this condition at the last chunk of a stream, where `response_metadata={'finish_reason': 'tool_calls'}`. At this point `_dict["tool_calls"]` is None, `default_class == AIToolCallsMessageChunk`, and `role` is None. This was a carry-over from when tool_calls was not optional, so I'll delete it for now. lmk if I'm misunderstanding anything.